bot.py

```diff
@@ -134,9 +135,27 @@ async def on_message(self, message):
         # Execute the command.
         await handler(message, command_content)
 
-    def next_in_queue(self, _):
+    def _stop(self):
+        """
+        voice_client.stop() triggers the after callback, so we need to set a state so it knows
```

Comment:
Instead of meta commentary on why this is needed, maybe just explain what the function does. For example the comment here could be "Stops playing music" and then you can have a comment in the constructor for the self.block_after variable and what it signals.

Reply:
and then the comment in the constructor of course

Reply:
Isn't `_stop` a helper function which stops playing music?

Reply:
Sounds good 👍